### PR TITLE
[Docs] Add guide for contributing `mesheryctl` reference

### LINKED ISSUE #6043: [Docs] Add section for contributing `mesheryctl` reference
#### Current State
3 months back, a CI action has been created to auto generate doc pages in mesheryctl reference section to avoid manual updates. But the newcomers are unaware of this and hence manipulate the markdown pages directly, which would eventually get erased by the action.

#### Desired State
A section must be created where users will understand about the new CI action thereby contributing to docs in proper manner

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR fixes #6043 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 35576be19d0b35925356439461c7d2ffab329b19 at: https://62fbb52e077e320e2f7b4225--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -248,20 +248,23 @@ Whenever the code tags are detected, the clipboard javascript file is automatica
...
 ## Documentation Contribution Flow Summary
 
+**NOTE: For contributing `mesheryctl` reference section, refer [Contributing CLI](/contributing-cli)**
+
+
 The following is a concise summary of the steps to contribute to Meshery documentation.
 
 1. Create a fork, if you have not already, by following the steps described [here](CONTRIBUTING-gitflow.md)
-1. In the local copy of your fork, navigate to the docs folder.
+2. In the local copy of your fork, navigate to the docs folder.
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:257]: Generally speaking, auto-numbering with `1.` is preferred.

File: docs/pages/project/contributing/contributing-cli.md [hunk 2]
```diff
@@ -140,6 +143,52 @@ Unit tests: Individual components are tested.
 
 Integration tests: Individual components are combined and tested as a group.
 
+# Contributing to mesheryctl documentation
+
+The documentation for `mesheryctl` is made different such that if the change is made on any of the `mesheryctl`'s golang files, the documentation page is created after the PR is merged, thereby reducing the extra work to update the documentation pages. To contribute to this section, follow the below steps:
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-cli.md:148]:
```suggestion
The [Meshery CLI Reference](/reference/mesheryctl) is autogenerated based on docs sections in each of  `mesheryctl`'s Golang files. Meshery CLI Reference pages are updated each time a change to its respective `mesheryctl` Golang file is merged into the project's `master` branch. This approach to documentation facilitates a single source of truth for CLI syntax and command behavior, which results in higher quality reference and a reduction in the toil involved in keeping documentation up-to-date. To contribute to the Meshery CLI Reference, follow these steps:
```

File: docs/pages/project/contributing/contributing-cli.md [hunk 3]
```diff
@@ -45,7 +45,10 @@ For logs, `mesheryctl` uses [Logrus](https://github.com/sirupsen/logrus). Going
 
 **Unit Tests**
 
-Unit test code coverage reports can be found - _tbd_.
+Unit test code coverage reports can be found in the [CodeCov logs](https://app.codecov.io/gh/meshery/meshery/).  (Note: GitHub login may be required for access)
+
+**Documentation**
+The documentation pages for `mesheryctl` reference are made with the help of `golang` script which will be triggered on PR merge using GitHub Actions. Refer to [Contributing mesheryctl documentation](#contributing-to-mesheryctl-documentation) for instructions.
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-cli.md:51]:
```suggestion
The documentation pages for `mesheryctl` reference are made with the help of the Cobra Golang framework and use of GitHub Actions. Refer to [Contributing to mesheryctl documentation](#contributing-to-mesheryctl-documentation) for details.
```

File: docs/pages/project/contributing/contributing-cli.md [hunk 4]
```diff
@@ -45,7 +45,10 @@ For logs, `mesheryctl` uses [Logrus](https://github.com/sirupsen/logrus). Going
 
 **Unit Tests**
 
-Unit test code coverage reports can be found - _tbd_.
+Unit test code coverage reports can be found in the [CodeCov logs](https://app.codecov.io/gh/meshery/meshery/).  (Note: GitHub login may be required for access)
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-cli.md:48]:
```suggestion
Unit test code coverage reports can be found in the [CodeCov logs](https://app.codecov.io/gh/meshery/meshery/).  _Note: GitHub login may be required for access._
```

File: docs/pages/project/contributing/contributing-docs.md (hunk 1 above)
Comment (alphaX86) [reply docs/pages/project/contributing/contributing-docs.md:257]: 👍

**Comment by l5io** [comment]
🚀 Preview for commit 7f5a3fff428aaa15008e6950928565ae1a8dc561 at: https://62fc495a6c29413c4471a46b--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Excellent 😊

**Pull request merged**